gc3libs/utils.py

```diff
@@ -1814,28 +1638,6 @@ class PlusInfinity(Singleton):
         >>> x > object()
         True
 
-    Note that `PlusInfinity` is a singleton, therefore you always get
-    the same instance when calling the class constructor::
-
-        >>> x = PlusInfinity()
-        >>> y = PlusInfinity()
-        >>> x is y
-        True
-
-    Relational operators try to return the correct value when
-    comparing `PlusInfinity` to itself::
-
-        >>> x < y
```

Comment:
Why remove doctests?

Reply:
Because it is no longer a Singleton, we are no longer comparing PlusInfinity to itself

Reply:
Well, but you still want to be sure that two instances of `PlusInfinity` implement that behavior.

Reply:
(I mean, if `x = PlusInfinity(); y = PlusInfinity()` then there's some expectation about how the two relate, and that contract is written in those tests.)

Reply:
OK, I reinstated the docs and checked that the examples hold. Still, see my comment above removing this class entirely.

However, the object promises to be larger than any object, but does not evaluate to being larger than an object of the same type. This doesn't make a great deal of sense, and I think indicates that the places where this class is used could probably be fixed to no longer need it...